Worksheet_B/README.md

```diff
@@ -0,0 +1,17 @@
+Print “ Welcome to Terminal 3 ”
+Print “ Password Required “
+Secret word = Pool
+Player strikes left: = 4
+Computer generates 14 words 4 letter words in random order
+Computer words = (“Pool, Roam, Seek, Dean, Foes, Half, Word, Copy, Cold, Fend, Wins, Pick, Died, Menu”)
+If player word is not equal to secret word player loses a strike
+	If letter = same position is Secret Word display similarity
+	If player word = Secret word minus 1 from player strikes
+
+Player chooses a word
+	If player word = Secret word player wins
+	If player word = Secret word
+Loop
+Print “ You win “
+
+![Flowchart](IMG_1497.jpg)
```

Comment:
Flowchart appears to only give the player one attempt -- needs a loop to allow multiple guesses, with decision nodes to break out of the loop if the correct word is guessed or if the number of allowed guesses is exhausted.